src/v2/components/BlockLightbox/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import { unescape } from 'underscore'
+
+import constants from 'v2/styles/constants'
+
+import { BlockLightbox as Block } from '__generated__/BlockLightbox'
+
+import Box from 'v2/components/UI/Box'
+import Title from 'v2/components/UI/Head/components/Title'
+import BlockLightboxContentPane from 'v2/components/BlockLightbox/components/BlockLightboxContentPane'
+import BlockLightboxMetadataPane from 'v2/components/BlockLightbox/components/BlockLightboxMetadataPane'
+import { SplitPane } from 'v2/components/UI/SplitPane'
```

Comment:
Just a note that avoiding default exports helps contain drift. With a default export, the person importing can name the variable whatever, where a named export ties things 1:1 to the object being exported, lending itself to static analysis, find and replace, maintenance over time, etc.

Reply:
Yeah, I've completely moved over personally as well.

Reply:
Will do, thanks for that info. I thought it was just a style thing.